### ISSUE TITLE
MO ID not created properly in iNat

### ISSUE THREAD

**Issue by JoeCohen** [JacobPulk/mirror#4]
FYI
https://www.inaturalist.org/observations/182722568
This mirrored Observation had an MO id of _Pterospora andromeda_, but the copied iNat Obs had an ID of 
[State of matter Life](https://www.inaturalist.org/taxa/48460).
No big deal, but I thought you'd like to know.

**Comment by JacobPulk** [comment]
Yeah, non-fungal/protozoan names are unsupported, I suppose because of
inter-kingdom homonyms - the program doesn't know if you really mean the
plant name, or there's a fungal homonym that's just missing from iNat. I
suppose it could be changed to assume that if the name is on iNat for any
kingdom, that's what was intended...

**Comment by JoeCohen** [comment]
Thanks Jacob!
If you wanted, you could support *Plantae*, as there should be no
inter-kingdom homonyms with *Fungi.*
Again, no big deal, as very few MO Obs are *Plantae* and it's easy to
change them manually on iNat after they are mirrored.

**Comment by JacobPulk** [comment]
Good point! I should do that. Thank you!
